src/countries/NL.json

```diff
@@ -22,19 +19,15 @@
 
     "TestData" : {
         "Max" : [ "9999ZZ" ],
-        "Min" : [ "0000AA" ],
-        "Valid" : ["1235DF", "5983DH"],
-        "Invalid" : [ "12j4h", "k3j51l" ],
+        "Min" : [ ],
```

Comment:
Would it be possible to have the Min `1000AA` here as you had in the previous commits, if it's not breaking anything? That would be my last comment :)

Reply:
@mmaruniak No, that generates a test case where the predecessor of postal code `1000AA` is checked. That predecessor throws an error, though, as the predecessor builder generates `0999ZZ`, which is an invalid postal code.

That's why that test case had to be removed: https://github.com/Cimpress-MCP/PostalCodes.Net/pull/74#discussion_r334885250